src/components/Media.jsx

```diff
@@ -10,7 +10,7 @@ class Media extends Component {
 
   async componentDidMount() {
     const token = await getToken();
-    Video.connect(token, { name: 'Room 1' })
+    Video.connect(token, { name: token.room })
```

Comment:
wait are you sure this worked? isn't token an object? shouldn't it have been token.token?